pyvista/core/filters/data_set.py

```diff
@@ -5213,3 +5218,128 @@ def integrate_data(self, progress_bar=False):
         alg.SetDivideAllCellDataByVolume(False)
         _update_alg(alg, progress_bar, 'Integrating Variables')
         return _get_output(alg)
+
+    def partition(self, n_partitions, generate_global_id=False, as_composite=True):
+        """Break down input dataset into requested number of partitions.
+
+        Cells on boundaries are uniquely to each partition.
```

Comment:
Aren't all cells unique to each partition?

In case this stays: `are uniquely` -> `are unique`.

Reply:
```suggestion
        Cells on boundaries are uniquely assigned to each partition without duplication.
```